oasislmf/computation/generate/files.py

```diff
@@ -418,9 +421,11 @@ def _validate_input_arguments(self):
 
     def _create_target_directory(self, label):
         utcnow = get_utctimestamp(fmt='%Y%m%d%H%M%S')
```

Comment:
function lacks docstring

Reply:
Its a good point, but the aim of the PR was to improve testing and fix issues raised. 

My preference would be to open a new issue/PR dedicated to adding missing doc-strings throughout the codebase